roles/solr_collection/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+# tasks file for roles/solr_collection
+- name: Create the base directory for Solr configsets
+  file:
+    path: "/solr/cdh_solr/"
+    state: directory
+    mode: 0755
+    owner: "{{ deploy_user }}"
+    group: "{{ deploy_user }}"
+  become: true
+  delegate_to: lib-solr-staging1.princeton.edu  # solr_staging
+
+- name: Update and configure Solr configset
+  block:
+    - name: Copy project Solr configset files for upload to zookeeper
+      # rsync configset from deployed version on remote host to solr server
+      become: true
+      become_user: "{{ deploy_user }}"
+      command: /usr/bin/rsync -a cdh-test-geniza1:{{ deploy }}/solr_conf/ /solr/cdh_solr/cdh_{{ app_name }}_solr_conf
+      delegate_to: lib-solr-staging1.princeton.edu  # solr_staging
+
+    - name: Update configset with zookeeper
+      command: "/opt/solr/bin/solr zk -upconfig -d /solr/cdh_solr/cdh_{{ app_name }}_solr_conf -n {{ solr_configset }} -z {{ zk_host }}"
+      delegate_to: lib-solr-staging1.princeton.edu
+
+    - name: Get a list of current Solr collections
+      uri:
+        url: "{{ solr_url }}admin/collections?action=LIST"
+      register: solr_collections_list
+
+    - name: Create Solr collection if it does not already exist
+      uri:
+        url: "{{ solr_url }}admin/collections?action=CREATE&name={{ solr_collection }}&collection.configName={{ solr_configset }}&numShards={{ num_shards }}&replicationFactor={{ replication_factor }}&maxShardsPerNode={{ shards_per_node }}"
+      when: solr_collection not in solr_collections_list.json.collections
```

Comment:
I don't see the for-loop where we would iterate through `solr_collections_list`, or where `solr_collection` is defined. Would someone mind explaining this to me?